src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -89,6 +91,7 @@ def logout(self):
         logger.debug('Logout successful')
         token = JwtManager.get_token_from_header()
         JwtManager.blocklist_token(token)
+        cherrypy.response.cookie['token'] = {'expires': 0, 'max-age': 0}
```

Comment:
Provided we do the same in  SAML controller, why not just encapsulate this in a protected method
(e.g. ```_delete_auth_token``` or sth similar) in BaseController and call it where appropriate?

Reply:
Done.